src/test/objectstore/store_test.cc

```diff
@@ -5901,7 +5909,9 @@ TEST_P(StoreTest, garbageCollection) {
       ASSERT_EQ(r, 0);
       ASSERT_EQ(statfs.compressed_allocated, 0x20000);
       const PerfCounters* counters = store->get_perf_counters();
+#if defined(HAVE_LIBAIO)
```

Comment:
@wjwithagen no in mean the whole test of `StoreTest.garbageCollection` is bluestore specific, you can guard it like
```c++
#ifdef HAVE_LIBAIO
TEST_P(StoreTest, garbageCollection) {
 // ...
}
```

the `#ifdef` versus `#if defined()` is but person preference. i am good either way.